examples/CMakeLists.txt

```diff
@@ -1,5 +1,6 @@
 cmake_minimum_required(VERSION 3.5)
-project(IsotropicWavelets-riesz_phase_analysis)
+# project(IsotropicWavelets-runWaveletCoeffsPhaseAnalyzisImageFilter_example)
```

Comment:
Here we want `project(IsotropicWaveletsExamples)`, or similar.

Typically, there is only a single invocation of the `project` command per CMake project.